src/main/resources/config.yml

```diff
@@ -80,6 +80,8 @@ traits:
   zoglin-baby: true
   piglin-baby: true
   leashed: true
+  # (**)
```

Comment:
It would be better to describe what the config path does, since its not obvious.